content/en/docs/reference/glossary/operator-controller.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Operator
+id: operator-controller
+date: 2019-05-21
+full_link: /docs/concepts/extend-kubernetes/operator/
+short_description: >
+  A controller used to manage a custom resource
+
+aka:
+tags:
+- architecture
+---
+ An operator is a {{< glossary_tooltip term_id="controller" >}} linked to one
+or more custom resources.
+
+<!--more-->
+
+[Operators](/docs/concepts/extend-kubernetes/operator/) are essentially a concept
+(and not a special kind of Kubernetes object).
```

Comment:
`Operators` is a concept defined by ...
or the `Operator pattern`
I think it makes more sense to describe as a pattern.